templates/jobservice/jobservice-dpl.yaml

```diff
@@ -90,6 +126,8 @@ spec:
         ports:
         - containerPort: {{ template "harbor.jobservice.containerPort" . }}
         volumeMounts:
+        - name: ca-certificates
+          mountPath: /tmp/ca-certificates
```

Comment:
Should probably be `/var/lib/ca-certificates`

Reply:
Right! This might be the main reason for my failures...